### DISCUSSION TITLE
Is there a way to turn the extension off temporarily?

### DISCUSSION THREAD

**Discussion by glauber-experian** [VSCodeVim/Vim#8379] [Q&A]
Wow, this really feels like Vim.  I have a quick question. Is there a keystroke or command to disable/enable "vim mode", in case i need to go back to vanilla VS Code?

Note: i found answers in Stack Overflow mentioning a command: "toggleVim", but it doesn't seem to exist in the current version.

Thanks,

glauber

**Comment by elzup** [comment]
I can found command at `v1.25.2` 
Command `Vim: Toggle Vim Mode`
Default keybinding is none

<img width="713" alt="Screenshot 2023-04-19 at 10 43 51" src="https://user-images.githubusercontent.com/2284908/232944922-667736d9-ed20-42d7-b61b-d432fcbf554f.png">

**Comment by OverRevvv** [comment]
### Yes, This is how you can set a hotkey to toggle this extension 

![image](https://user-images.githubusercontent.com/98686837/233698243-1d59cda9-c851-4b8b-9796-c46a49899329.png)


- Step 1: Press  Ctrl + P, Open Command palette
- Step 2: Search " Toggle Vim"
- Step 3: Add a keybinding 

## Done!!

**Comment by RevNexS** [comment]
Is there way to have shortcut like ctrl+; or anything else to disable and enable vim?
Or how can I create custom shortcut for it I tried following in keybindings.json ().
```
[
  {
    "key": "ctrl+;",
    "command": "vim.toggleVim", 
  }
]
```
This didnt worked for me.

**Reply by OverRevvv** [reply]
Did you tried any other key combination?

```json
    {
        "key": "alt+v",
        "command": "toggleVim"
    },
````

I use this combination to turn it on or off, give it a try and other thing remove `vim.` from "vim.toggleVim",

**Reply by RevNexS** [reply]
Thanks it worked
